.cargo/config.toml

```diff
@@ -1,2 +1,3 @@
 [env]
-RUST_TEST_THREADS = "1"
\ No newline at end of file
+RUST_TEST_THREADS = "1"
+SOCKET_DIR = { value = "/run/aziot", force = false }
```

Comment:
Where does the SOCKET_DIR variable come from and why do we need it at build time?

Reply:
it come from eis-utils dependency which now makes use of azure iot-identity-service 1.5.0 which requires that compile time envvar

Reply:
I see, can we maybe add a comment so that we don't forget about that?

Reply:
done